HW03/yuliya_sheremet/yuliya_sheremet_hw_03_t_03.rb

```diff
@@ -0,0 +1,49 @@
+# HW03/task3.rb
+# frozen_string_literal: true
+
+require 'rubocop'
+require 'date'
+require 'time'
+
+class Homework3
+  def task3(log)
+    array_of_strings = log.split(/\n/)
+    cases_for_output(log, array_of_strings)
+  end
+
+  def cases_for_output(logs, array_of_strings)
+    if logs.scan('Calling core').length > 1
+      if_include(array_of_strings)
+      seconds = array_of_strings
+      check_and_print(seconds)
```

Comment:
we usually don't use `and` for method name, looks like `check` is not necessary here
```suggestion
      print(seconds)
```